src/atoms/Calendar/Dropdown.tsx

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import 'isomorphic-fetch';
+import React from 'react';
+import TextField from '@material-ui/core/TextField';
+import Autocomplete from '@material-ui/lab/Autocomplete';
+import CircularProgress from '@material-ui/core/CircularProgress';
+
+interface CourseType {
+  subject: string;
+  code: string;
+}
+
+const Dropdown: React.FC = () => {
+  const [open, setOpen] = React.useState(false);
+  const [options, setOptions] = React.useState<CourseType[]>([]);
+  const loading = open && options.length === 0;
+
+  // TODO: Save the courses added by a user
+  /**
+  Adds course to user.
+  @param {any} _event The onChange event.
+  @param {any} num2 The value of the input.
+  @returns {any} The course that has been added.
+ */
+  function handleAddCourse(_event: any, value: any) {
+    return value;
+  }
+
+  React.useEffect(() => {
+    let active = true;
+
+    if (!loading) {
+      return undefined;
+    }
+
+    (async () => {
+      // TODO: Import the user's active termCode this is a placeholder
+      const termCode = 201901;
+      const response = await fetch(`https://uvic-scheduler-server.herokuapp.com/courses?term=${termCode}`);
+      const courses = await response.json();
+      const space = ' ' as any;
+
+      if (active) {
+        setOptions(Object.keys(courses).map(key => courses[key].subject + space + courses[key].code) as CourseType[]);
```

Comment:
You can use a template string instead. `${courses[key].subject} ${courses[key].code}`
Doesn't need to be changed but I find this easier to read

Reply:
When I try to use the string it says it may not be compatible with CourseType[] but I'll take another look to see if I can use it by changing the type!